script.bluetooth.delay/addon.py

```diff
@@ -99,24 +79,32 @@ def speakers():
 def main():
 
     if arg == "reset":
-        xbmc.executebuiltin('PlayerControl(stop)')
-        Addon.setSettingBool('state', 1)
-        reset1()
+        if (xbmc.getCondVisibility('Player.HasVideo') == False):
+            xbmc.executebuiltin("PlayMedia("+path+"reset.mp4,1)")
+        xbmc.executebuiltin('ActivateWindow(busydialognocancel)')
+        time.sleep(2)
+        for x in range (800):
+            xbmc.executebuiltin("Action(AudioDelayPlus)")
+        for x in range (400):
+            xbmc.executebuiltin("Action(AudioDelayMinus)")
+        time.sleep(1)
         skin1()
-        xbmcgui.Dialog().notification("",Addon.getLocalizedString(30006), "",t*2)
+        time.sleep(1)
+        xbmc.executebuiltin('Dialog.Close(busydialognocancel)')
+        Addon.setSettingBool('state', 1)
```

Comment:
Why are you storing state in settings? Isn't it scoped to the running script? State should be a class variable or similar

Reply:
Hello enen92
Need the state in the settings, so that it is saved even after restarting kodi.